docs/source/getting_started/user_guide.md

```diff
@@ -40,27 +51,27 @@ class RandomClassificationDataset:
 
 There are a few parameters that need to be initialized before {class}`streaming.MDSWriter` gets called. Some of the parameters are optional, and others are required parameters. Let's look at each of them where we start with two required parameters.
 
-1. Provide the Local filesystem directory path to store the compressed dataset
+1. Provide the Local filesystem directory path to store the compressed dataset files.
```

Comment:
local filesystem

Reply:
Done in https://github.com/mosaicml/streaming/pull/33/commits/a634dbdbd8039c0755687d3cc8172cba15c977ed.